codebuild_specs/mock_e2e_tests.yml

```diff
@@ -7,4 +7,7 @@ env:
 phases:
   build:
     commands:
-      - source ./shared-scripts.sh && _mockE2ETests
\ No newline at end of file
+      - source ./shared-scripts.sh && _mockE2ETests
+artifacts:
```

Comment:
Is this added because the build is a parent of another build? If that is the case, I think we need some improvement on the artifact upload if it is only `shared-script.sh` as we are not taking a good usage of s3 bucket